Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt

```diff
@@ -105,14 +107,17 @@ fun ConversationContent(
                             Message(authorMe, content, timeNow)
                         )
                     },
-                    scrollState
+                    scrollState = scrollState,
+                    modifier = Modifier.navigationBarsWithImePadding(),
```

Comment:
Add a comment here? similar to 
`// Use statusBarsPadding() to move the app bar content below the status bar`